### PR TITLE
Add scheduled_releases.yml workflow

### PR DESCRIPTION
**Pull request by colin-rogers-dbt**
Schedule the release of bundles

### CODE REVIEW

**Comment by colin-rogers-dbt** [comment]
Successful integ tests: https://github.com/dbt-labs/dbt-core-snapshots/actions/runs/4378854587

**Review by emmyoop — CHANGES_REQUESTED** [review]
You're going to have to bypass the checks to get this merged in.  it runs the workflow on `main` which is why the test are failing.

File: test/integration/snapshot/test_create.py [hunk 1]
```diff
@@ -10,9 +12,14 @@
...
     created_assets = generate_snapshot(Version.coerce(test_version))
-    for asset_name, asset_location in created_assets:
+    for asset_name, asset_location in created_assets.items():
         assert os.path.isfile(asset_location)
         if ".zip" in asset_name:
             assert zipfile.is_zipfile(asset_location)
-        else:
-            assert os.path.getsize(asset_location) > 0
+        assert os.path.getsize(asset_location) > 0
+        if ".txt" in asset_name:
```
[the remark this answers is not in the document]
Comment (colin-rogers-dbt) [reply test/integration/snapshot/test_create.py:20]: I realized I should check _any_ file for being empty not just .txt

File: .github/workflows/scheduled_releases.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  schedule:
+    - cron: "0 13 * * 1"
+  workflow_dispatch:
+
+jobs:
+  release-bundles:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+       version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.4.0b1", "1.5.0b1"]
```
[the remark this answers is not in the document]
Comment (colin-rogers-dbt) [reply .github/workflows/scheduled_releases.yml:29]: Pretty much,`1.4.0b1` maps to the 1.4 pre release that we need in cloud. 

Added a matrix comment. I'm going to follow this pr up with a pr adding documentation explaining what to do when we add a new dbt version

File: test/integration/snapshot/test_create.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmyoop) [reply test/integration/snapshot/test_create.py:23]: Why not move the empty check to before the zip check?  Also, do you want to fail if it's not zip or txt.  Or that expected?

[the remark this answers is not in the document]
Comment (colin-rogers-dbt) [reply test/integration/snapshot/test_create.py]: good point, cleaned up and added an else condition to raise if not txt or zip

**Review by emmyoop — APPROVED** [review]

**Pull request merged**